tests/allowed.json

```diff
@@ -0,0 +1,41 @@
+[
```

Comment:
From my experience this file can change randomly when running on different devices.
I suggest this: https://symfony.com/doc/current/components/phpunit_bridge.html#ignoring-deprecations

Reply:
The `ignoreFile` option is only available since Symfony `6.1`!

Reply:
Ah, sad :(